recipes/wavelet_buffer/all/conanfile.py

```diff
@@ -0,0 +1,125 @@
+    requires = "openblas/0.3.20", "blaze/3.8", "libjpeg-turbo/2.1.2", "cimg/3.0.2"
+    default_options = {
+        "cimg:enable_fftw": False,
+        "cimg:enable_jpeg": False,
+        "cimg:enable_openexr": False,
+        "cimg:enable_png": False,
+        "cimg:enable_tiff": False,
+        "cimg:enable_ffmpeg": False,
+        "cimg:enable_opencv": False,
+        "shared": False,
+        "fPIC": True,
+    }
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+
+    generators = "CMakeDeps"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+        self.settings.build_type = "Release"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        get(
+            self,
+            **self.conan_data["sources"][self.version],
+            destination=self.source_folder,
+            strip_root=True
+        )
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["CONAN_EXPORTED"] = True
+        tc.generate()
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 20
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "19",
+            "gcc": "8",
+            "clang": "13",
+            "apple-clang": "13",
+        }
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._minimum_cpp_standard)
+
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn(
+                "{} recipe lacks information about the {} compiler support.".format(
+                    self.name, self.settings.compiler
+                )
+            )
+        else:
+            if Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(
+                    "{} requires C++{} support. The current compiler {} {} does not support it.".format(
+                        self.name,
+                        self._minimum_cpp_standard,
+                        self.settings.compiler,
+                        self.settings.compiler.version,
+                    )
+                )
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(
+            self,
+            pattern="LICENSE",
+            dst=os.path.join(self.package_folder, "licenses"),
+            src=self.source_folder,
+        )
+
+        cmake = CMake(self)
+        cmake.install()
+
+        # some files extensions and folders are not allowed. Please, read the FAQs to get informed.
```

Comment:
Please, only keep remove method for what is produced by the project. Avoid running unnecessary methods. Only see cmake files being generated: https://github.com/panda-official/WaveletBuffer/blob/v0.4.0/CMakeLists.txt

Reply:
Done